### DISCUSSION TITLE
"Inherited access" not checked and package setting not visible

### DISCUSSION THREAD

**Discussion by mbenzekri** [community/community#37320] [Enterprise]
Hi,
We are experiencing new issue on github concerning package access, visibility and administration. 
When I publish a new package using “npm publish” (I am owner and Admin on repo), 

1. Package is visible by me but not visible by other repo users
2. Access is not inherited from repo by default (“Inherited access” not checked)
3. I **do not have access to “package setting” to change “Inherited Access”**

After asking our Enterprise account owner to check “Inherited access” in “package settings” all 3 listed problems are solved:

1. Package is visible by all repo users
2. “Inherited access” checked (done by account owner)
3. I have access to “package settings”

What am-i doing wrong, to not having access to “package settings” ? 
I would like to avoid asking account owner to change that checkbox at each package creation ?

**Comment by ossanna16** [comment]
Hi there @mbenzekri and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
